### ISSUE TITLE
[Bug][A11Y] Chip: Hint is wrong on Android and missing on iOS

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/ouds-flutter#393]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-flutter/issues) for duplicate or closed feature requests

### Your test device

iOS and Android

### OS Version

.

### App version

0.6.0

### Describe the issue

**On Android:** The hint on a selected chip state is "_double-tap to activate_" instead of "_double-tap to unselect_" (or _deactivate_ but _unselect_ is maybe more relevant for that kind of component)

**On iOS:** The hint is missing for both states unselected and selected

### Expected Behavior

The hint should be "_double-tap to select/unselect_" or "_double-tap to toggle setting_"

**Comment by pya35** [comment]
the best thing to do would be to announce the current state "unselected" or "selected" then the label and finally the hint.

**Comment by nouha06** [comment]
@pya35 

- On Android, the screen reader automatically announces “double-tap to activate” because the OS detects that the component is selectable or actionable. This message is generated by the system based on accessibility traits, not by Flutter, and simply indicates that the widget can be interacted with. In this case, we should provide a personalized accessibility label and remove the selected state to prevent the reader from announcing “double-tap to activate”.

-  On iOS, however, even when the OS detects that the component is selectable, it does not automatically announce “double-tap to activate.

**Comment by Tayebsed93** [comment]
The issue has been fixed and the ticket has been closed, as we cannot release a new version. That’s why the TestFlight blocking label was applied.

**Comment by B3nz01d** [comment]
Closed during Sprint # 23

**Comment by nouha06** [comment]
📣 New Firebase App Distribution BETA upload available 🚀 (Android)

Please, get the new build from App Tester app, or contact the project maintainers (@Tayebsed93) to get access.

Display name: DesignToolbox (BETA) (485)
Version: 0.6.0
Build type: BETA
Build details (GitHub): 443
cc @B3nz01d (product owner)
cc @Tayebsed93 @AhmedAmineZr @nouha06 (dev team)
cc @jerome-regnier (design team)
cc @pya35 (accessibility team)

**Comment by nouha06** [comment]
New TestFlight BETA upload available 🚀 (iOS)

Please, get the new build from TestFlight app, or contact the project maintainers (@Tayebsed93) to get access.

Display name: Design Toolbox Flutter (485)
Version: 0.6.0
Build type: BETA
Build details (GitHub): 443
cc @B3nz01d (product owner)
cc @Tayebsed93 @AhmedAmineZr @nouha06 (dev team)
cc @jerome-regnier (design team)
cc @pya35 (accessibility team)
